packages/components/src/internal/components/user/UserLink.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { List } from 'immutable';
+import { User as IUser } from '@labkey/api'
+
+import { AppURL, User } from '../../..';
+
+interface Props {
+    currentUser: User
+    allUsers: List<IUser>
+    userId: string
+}
+
+export class UserLink extends React.PureComponent<Props, any> {
```

Comment:
minor (I know this was existing code copied over): could remove the `any` for the state prop as it isn't needed.